### ISSUE TITLE
Neuer Formelbereich

### ISSUE THREAD

**Issue by helfjama** [iqb-berlin/verona-modules-aspect#711]
Anforderungen:
- verwendet x statt Malpunkt
- kein automatisches ein Bruchgenerierten 
- Leerzeichen im Formeleditor und Enter-Taste funktionieren nicht - Recherche?
- "Formel einfügen" als Label
- Scroll into View

Anforderungen an die Tastatur:
- Shift bei "ABC"+ griech. Zeichen, erzeugt ein Mülleimer-Symbol - weg
- vertikale Pfeilnaviagation
- beim Mouseover "Römische" und "Griechische" wird kleingeschrieben

**Comment by rhenck** [comment]
"Leerzeichen im Formeleditor funktioniert nicht"

Leerzeichen haben in einer Formel auch nix zu suchen oder wie?!

**Comment by rhenck** [comment]
"Enter-Taste funktioniert nicht - rausmachen?"

Was wäre das erwartete Verhalten? Zeilenumbrüche haben ebenfalls nix in einer Formel zu suchen.

**Comment by helfjama** [comment]
> "Enter-Taste funktioniert nicht - rausmachen?"
> 
> Was wäre das erwartete Verhalten? Zeilenumbrüche haben ebenfalls nix in einer Formel zu suchen.

Ja, sehe ich genauso - Enter-Taste dann rausnehmen? Das Verhalten sie zu haben ohne dass sie was macht ist ja komisch?

**Comment by helfjama** [comment]
> "Leerzeichen im Formeleditor funktioniert nicht"
> 
> Leerzeichen haben in einer Formel auch nix zu suchen oder wie?!

Leerzeichen hingegen wären praktisch, aber ja, Latex erlaubt die glaube ich nicht? Du kannst ja mal schauen was sich machen lässt?
